### ISSUE TITLE
pyro: allow subscriptions from the outside in

### ISSUE THREAD

**Issue by hosted-fornet** [uqbar-dao/uqbar-core#364]
Do a similar thing as we did with the poke primitive, e.g., for gall replace the %raw-poke in the [deal](https://github.com/urbit/urbit/blob/master/pkg/arvo/sys/lull.hoon#L1817-L1819) with a [%watch](https://github.com/urbit/urbit/blob/master/pkg/arvo/sys/lull.hoon#L1841-L1847).

Will require more infrastructure work to pass messages from inside %pyro to the outside world (h/t dachus).

The particular use-case that prompted this is wanting to subscribe to clay on a virtualship so that we can do something like:
1. Sync desk from devship to virtualship,
2. Wait until sync is done, by waiting on a subscription update from clay,
3. Install the virtualship desk.

This is a low priority: would be cool but not blocking anything.

**Comment by tadad** [comment]
just looked into getting this done via gall's verbose mode (`|pass [%g %spew %odd ~]`) and it's not enough.
Main problem is that no effect is generated if a subscription %fact goes to nowhere - we can't fish it out without modifying arvo.

**Comment by tadad** [comment]
turns out this isn't an important feature
